addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -371,6 +364,22 @@ export class ProductScreen extends Component {
             : list.sort((a, b) => a.display_name.localeCompare(b.display_name));
     }
 
+    getProductsBySearchWord(searchWord) {
```

Comment:
why would you have a new function for this?

Reply:
Thanks for your review @vlst-odoo 
I wanted to make the code cleaner.